docker-compose-local.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.7"
+services:
+  jupyter-ml:
+    image: simcore/services/dynamic/${IMAGE_TO_RUN}:${TAG_TO_RUN}
+    ports:
+      - 8888:8888
+    environment:
+      - DY_SIDECAR_PATH_INPUTS=/tmp/inputs
+      - DY_SIDECAR_PATH_OUTPUTS=/tmp/outputs
+      - DY_BOOT_OPTION_BOOT_MODE=0
+      - DY_SIDECAR_PATH/home/jovyan/work/workspace
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix
```

Comment:
Again, I doubt we need an X server (smash does though)

Reply:
let me try and drop it, see what happens